pwndbg/glibc.py

```diff
@@ -17,10 +17,21 @@
 
 safe_lnk = pwndbg.config.Parameter('safe-linking', 'auto', 'whether glibc use safe-linking (on/off/auto)')
 
+glibc_version = pwndbg.config.Parameter('glibc', '', 'GLIBC version for heuristics', scope='heap')
+
 
 @pwndbg.proc.OnlyWhenRunning
-@pwndbg.memoize.reset_on_objfile
 def get_version():
+    if glibc_version.value:
+        ret = re.search(r"(\d+)\.(\d+)", glibc_version.value)
+        if ret:
+            return tuple(int(_) for _ in ret.groups())
+    return _get_version()
+
+
+@pwndbg.proc.OnlyWhenRunning
+@pwndbg.memoize.reset_on_objfile
+def _get_version():
```

Comment:
I think it may not hurt to do have reset_on_start too just in case someone sets some env options in GDB and so change the libc that would be loaded with the binary

Reply:
Ok, I'll add that event, too.